### DISCUSSION TITLE
Hiding the cursor

### DISCUSSION THREAD

**Discussion by simonmandlik** [nvim-tree/nvim-tree.lua#1159] [Q&A] [answered]
Hi, I have written this about 6 months ago:

```vim
autocmd BufWinEnter NvimTree setlocal cursorlineopt=line guicursor+=n:Cursor
autocmd BufEnter,FileType NvimTree call NvimTreeHideCursor()
autocmd BufLeave,BufWinLeave,WinClosed NvimTree call NvimTreeShowCursor()
function! NvimTreeHideCursor()
    highlight! Cursor blend=100
endfunction
function! NvimTreeShowCursor()
    highlight! Cursor blend=NONE
endfunction
```

It hides the cursor whenever the nvim-tree window is in the focus, because I find it quite distracting. 

I have recently updated all packages after quite a while and the cursor doesn't disappear.

Any idea why or what is the correct way to do it now?

**Comment by kyazdani42** [comment, marked as answer]
There are now multiple nvim tree window, replace NvimTree with `NvimTree*` :)

**Reply by simonmandlik** [reply]
Thanks!
